### ISSUE TITLE
Add a features table for all the different authentication packages

### ISSUE THREAD

**Issue by SuperDJ** [laravel/ideas#2491]
To help users make a decision on which authentication package to choose it would help to have a features table in the docs. Since there are so many authentication packages available now it would ease the decision making. 

Feature | UI | Breeze | Passport | Fortify | Jetstream |
| -           | -   | -          | -              |  -        |  -              |
| API support |-  | - | - | x | x |
| Password reset | x | - | - | - | - |

etc.

**Comment by rvanbaalen** [comment]
Yes, please. It's very confusing to compare one with the others and to actually pick a package.

**Comment by SuperDJ** [comment]
@rvanbaalen from what I understand

- [Laravel UI](https://github.com/laravel/ui)
  - "Old"/ legacy authentication package.
  - Login
  - Registration
  - Email validation
  - [Bootstrap](https://getbootstrap.com/) for UI 
- [Fortify](https://laravel.com/docs/8.x/fortify)
  - New "basic" authentication package
  - Login
  - Registration
  - Email verification
  - Password confirmation
  - Password rest
  - 2FA
  - No UI so no need to override it either
- [Breeze](https://github.com/laravel/breeze)
  - Replacement of Laravel UI 
  - Login
  - Registration
  - Password reset 
  - Email verification
  - Password confirmation
  - [Tailwind](https://tailwindcss.com/) for UI
- [Sanctum](https://laravel.com/docs/8.x/sanctum)
  - Basic API-authentication
  - For SPA
- [Passport](https://laravel.com/docs/8.x/passport)
  - Extended API-authentication
  - OAuth support
  - Service-to-service/ machine-to-machine support
- [Jetsream](https://jetstream.laravel.com/2.x/introduction.html)
  - Uses Fortify under the hood so all of those functions
  - Uses Sanctum under the hood so all of those function
  - Uses Tailwind with [Livewire](https://laravel-livewire.com/) or Tailwind with [Inertia](https://inertiajs.com/) and Vue for UI
  - Includes teams support. This could be included in the other packages with a package like [Spatie permissions](https://spatie.be/docs/laravel-permission/v4/introduction)

These aren't all features of the packages but I hope I have made the differences a bit clearer.
